projects/payment-lib/src/lib/components/payment-view/payment-view.component.html

```diff
@@ -90,7 +90,7 @@ <h1 class="heading-large govuk-!-margin-top-0">{{ "Payment details" | rpxTransla
           </tr>
 
           <tr *ngIf="paymentGroup?.payments[0].organisation_name">
-            <td class="bold tb-col-w">{{ "PBA account name" | rpxTranslate" {"pba": "PBA"} }}</td>
+            <td class="bold tb-col-w">{{ "PBA account name" | rpxTranslate: {"pba": "PBA"} }}</td>
```

Comment:
Ben should it be?
<td class="bold tb-col-w">{{ "PBA account name" | rpxTranslate : {pba: "PBA"} }}</td>